docs/data/material/customization/palette/UsingStylesUtils.js

```diff
@@ -9,6 +9,10 @@ import Button from '@mui/material/Button';
 import { Box, Stack } from '@mui/system';
 import { Typography } from '@mui/material';
 
+// Augment the palette to include a violet color
+
+// Update the Button's color options to include a violet option
+
```

Comment:
These are comments to explain module augmentation from the `.tsx` example, is there a way to avoid them?

Reply:
Not that I'm aware of, we could add a TypeScript prefix to the comment, but I suspect it's not worth the effort.